2004-11-24  H.J. Lu  <hongjiu.lu@intel.com>

	PR 574:
	* elfcode.h (elf_slurp_symbol_table): Handle STT_TLS.

Index: bfd/elfcode.h
===================================================================
RCS file: /cvs/src/src/bfd/elfcode.h,v
retrieving revision 1.58
retrieving revision 1.59
diff -u -r1.58 -r1.59
--- bfd/elfcode.h	15 Nov 2004 23:21:24 -0000	1.58
+++ bfd/elfcode.h	25 Nov 2004 02:14:10 -0000	1.59
@@ -1158,6 +1158,9 @@
 	    case STT_OBJECT:
 	      sym->symbol.flags |= BSF_OBJECT;
 	      break;
+	    case STT_TLS:
+	      sym->symbol.flags |= BSF_THREAD_LOCAL;
+	      break;
 	    }
 
 	  if (dynamic)
